src/bindings/interface/annotator.i

```diff
@@ -0,0 +1,217 @@
+%module(package="libcellml") annotator
+
+#define LIBCELLML_EXPORT
+
+%import "createconstructor.i"
+%import "enums.i"
+%import "logger.i"
+%import "types.i"
+
+%include <std_pair.i>
+%include <std_vector.i>
+
+%feature("docstring") libcellml::Annotator
+"Interacts with CellML objects using their id attribute.";
+
+%feature("docstring") libcellml::Annotator::setModel
+"Set the model for this Annototor to work with.";
+
+%feature("docstring") libcellml::Annotator::item
+"Return the item with the given id.";
+
+%feature("docstring") libcellml::Annotator::component
+"Return the component with the given id.";
+
+%feature("docstring") libcellml::Annotator::encapsulation
+"Return the model with the given encapsulation id.";
+
+%feature("docstring") libcellml::Annotator::model
+"Return the model assigned to this Annotator or the model with the given id.";
+
+%feature("docstring") libcellml::Annotator::variable
+"Return the variable with the given id.";
+
+%feature("docstring") libcellml::Annotator::importSource
+"Return the import source with the given id.";
+
+%feature("docstring") libcellml::Annotator::units
+"Return the units with the given id.";
+
+%feature("docstring") libcellml::Annotator::reset
+"Return the reset with the given id.";
+
+%feature("docstring") libcellml::Annotator::testValue
+"Return the ResetPtr whose test_value has the given id.";
+
+%feature("docstring") libcellml::Annotator::resetValue
+"Return the ResetPtr whose reset_value has the given id.";
+
+%feature("docstring") libcellml::Annotator::componentRef
+"Return the ComponentPtr with the given component_ref id.";
+
+%feature("docstring") libcellml::Annotator::unit
+"Return the UnitsPtr and index whose Unit is with given id.";
+
+%feature("docstring") libcellml::Annotator::connection
+"Return a pair of VariablePtrs that define a connection with given id.";
+
```

Comment:
Ditto here - doesn't it now return a `VariablePair` item rather than two `VariablePtr`s?